repositories.bzl

```diff
@@ -0,0 +1,559 @@
+def boringssl_repositories(bind=True):
+    native.git_repository(
+        name = "boringssl",
+        commit = "bfd36df3da38dbf8828e712f42fbab2a0034bc40",  # 2017-02-02
+        remote = "https://boringssl.googlesource.com/boringssl",
+    )
+
+    if bind:
+        native.bind(
+            name = "boringssl_crypto",
+            actual = "@boringssl//:crypto",
+        )
+
+        native.bind(
+            name = "libssl",
+            actual = "@boringssl//:ssl",
+        )
```

Comment:
I know that I started this, but could we get rid of all those unnecessary binds now that Bazel has concept of named workspaces? They don't seem to provide any value in Envoy's case, and we can re-add them later (if we need to link to gRPC or something else).

(applies to the whole file)